GCTC-NTGC/gc-digital-talent
# Bug - 4313 - Render new lines text area

Resolves #4313 

## 📋 Summary

This wraps the content coming from a multi-line input (`textarea`) with a component that displays new lines with css.

## ❓ Question

While reviewing this, if you know of anywhere else we are using `textarea` and displaying the values, can you let me know so it doesn't get missed? 😄 

> **Note**
> I purposefully did not include this on the descriptions for pools in the `PoolCard`. Thought that may look strange 🤔 
> If you disagree, let me know and we can add it there as well.

## 🧪 Testing

1. Build all projects `npm run production --workspaces`
2. Edit a pool, adding new lines in "Your impact" and "Work tasks"
3. View the pool on admin and confirm line breaks appear
4. View the pool on "Browse Opportunities" confirm line breaks appear
5. Edit an experience on the profile
6. Added line breaks in skill descriptions and additional information
7. View the skills in all both Experience and Skill accordions confirm line breaks appear
